lib/oxidized/hook/githubrepo.rb

```diff
@@ -0,0 +1,16 @@
+class GithubRepo < Oxidized::Hook
+  def validate_cfg!
+    cfg.has_key?('remote_repo') or raise 'remote_repo is required'
```

Comment:
Raise an exception instead of a string? What's the error behaviour of the method being overriden?